nss-ice-phantoms/WorkforceManagement
# Cb departments 2

# Description

Got the department details view to list all the employees properly.

## Type of change

# Testing Instructions

1. `git fetch --all`
1. `git checkout cb-departments-2`
1. Run BangazonWorkforce
1. Verify that the department details view renders the employees for each dept

# Checklist:
